src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -1,8 +1,17 @@
 package com.minecolonies.coremod.blocks;
 
+import com.minecolonies.coremod.blocks.cactus.*;
+import com.minecolonies.coremod.blocks.decorative.*;
+import com.minecolonies.coremod.blocks.huts.*;
+import com.minecolonies.coremod.blocks.schematic.BlockSolidSubstitution;
+import com.minecolonies.coremod.blocks.schematic.BlockSubstitution;
+import com.minecolonies.coremod.blocks.schematic.BlockWaypoint;
+import com.minecolonies.coremod.blocks.types.TimberFrameType;
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockPlanks;
 import net.minecraft.item.Item;
+import net.minecraft.item.ItemBlock;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.item.ItemBlock'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)